x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.tsx

```diff
@@ -33,6 +37,15 @@ export const TrustedAppsPage = memo(() => {
     history.push(getTrustedAppsListPath(paginationParamsOnly));
   }, [history, urlParams]);
 
+  const backButton = useMemo(() => {
+    if (routeState && routeState.onBackButtonNavigateTo) {
+      return <BackToExternalAppButton {...routeState} />;
+    }
+    return null;
+    // FIXME: Route state is being deleted by some parent component
```

Comment:
this is to keep the route state from being deleted?  just understanding what's going on with this

Reply:
I don't know what is going on and have it my notes to open an issue for follow up. I have seen this before with one of our other pages.

When we arrive on this page, `state` is there, but then quickly (instantly) after it, it get deleted. I think it has to do with some ancestor component updating the the `history` from react router dropping state. So this was a work-around to ensure we don't lose it.